app/views/idv/jurisdiction_errors/no_id.html.erb

```diff
@@ -0,0 +1,13 @@
+<% title t("idv.titles.no_id") %>
+
+<%= image_tag(asset_url('alert/fail-x.svg'), width: 54) %>
```

Comment:
Do the accessibility tests cover this page? This should be reported as an error, as the image lacks a required `alt` attribute.

https://dequeuniversity.com/rules/axe/3.5/image-alt?application=AxeChrome
https://www.w3.org/TR/WCAG20-TECHS/F65.html

It could fit the criteria of [decorative image](https://www.w3.org/WAI/tutorials/images/decorative/), in which case it could be fixed by:

```suggestion
<%= image_tag(asset_url('alert/fail-x.svg'), width: 54, alt: '') %>
```

Reply:
Good catch, willfix.